libs/httpext/src/lib/http-ext.ts

```diff
@@ -17,9 +19,9 @@ export class HttpExt {
     request,
     handler
   }: {
-    request: Request<unknown>;
-    handler: (request: Request<unknown>) => Observable<any>;
-  }) {
+    request: Request;
+    handler: RequestHandlerFn;
```

Comment:
With some refactoring, the handler has the same signature as the `NextFn`.
I thought it would be easier to merge them.